docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -126,19 +206,21 @@ public class NoOpSingleStringFunction implements StringFunction {
     }
 }
 ----
-
-Quarkus also provides support for injecting configuration values using Spring's `@Value` annotation.
-To see that in action, first edit the `src/main/resources/application.properties` with the following content:
-
-[source,properties]
++
+//. Quarkus also provides support for injecting configuration values using Spring's `@Value` annotation.
```

Comment:
For Prs in general. Please Just what is changed rather than comment out. 

Much easier to review/see the delta when old is removed/changed in place.

Reply:
I was using the comments here to have a side-by-side comparison of what I am rewriting vs. the original wording, but I agree that too many long comments tend to mess up the diff view, so I am going to remove the commented out section before submitting rewrites for review for upcoming PRs